main/docs/versions/future-android.md

```diff
@@ -31,14 +31,29 @@ OsmAnd releases several Ready-To-Use builds daily, allowing you to track new fea
 
 # What's new
 
-- Buttons Control on map (Allow to change size, shape, icon for map buttons),
-- Context menu for widgets,
-- [Autosync](https://github.com/osmandapp/OsmAnd/issues/18212) OsmAnd Cloud,
-- [Smartwatch](https://github.com/osmandapp/OsmAnd/issues/9258),
-- Gallery / Places - Explore ([images on the map](https://osmand.net/docs/versions/future-web#explore-map-menu)),
-- Main navigation widgets,
-- [Activities](https://github.com/osmandapp/OsmAnd/issues/17997) for GPX track data,
-- Search (USA address),
-- Nautical maps (new data),
-- Traffic (Germany and Netherlands),
-- Geographic grid,
\ No newline at end of file
+- Buttons Control on map ([Allow to change size, shape, icon for map buttons](https://github.com/osmandapp/OsmAnd-Issues/issues/2437)).
+  - Docs
+- Context menu for widgets.
+  - Docs
+- [Autosync](https://github.com/osmandapp/OsmAnd/issues/18212) OsmAnd Cloud.
+  - Docs
+- [Smartwatch](https://github.com/osmandapp/OsmAnd/issues/9258).
+  - Docs
+- Gallery / Places - Explore ([images on the map](https://osmand.net/docs/versions/future-web#explore-map-menu)).
+  - Docs
+- [Key “D”, “N”, “S” on external Keyboards - no effect any more](https://github.com/osmandapp/OsmAnd/issues/20636)
```

Comment:
It's faster to add a meaningful single line like:
- Created quick actions to represent 'Map rotate', ... and 4.8 issue  ``` Key “D”, “N”, “S”  doesn't work on external Keyboards ``` is fixed